### ISSUE TITLE
The button props for color not working properly.

### ISSUE THREAD

**Issue by kashua14** [jonatanklosko/material-ui-confirm#66]
I wanted to change the color for the cancel and confirm buttons but it did not work as I had hoped. Could you guide me on how best I can fix this issue? because I think am doing it wrong.
```js
cancellationButtonProps: { color: "warning" },
confirmationButtonProps: { color: "error" },
```

The `primary & secondary` color prop values work but the other color props for the MUI Button aren't working.

I know this does not sound like an issue but I had no other medium I could use to get your help.

**Comment by jonatanklosko** [comment]
Hey @kashua14! Are you using the latest Material UI (5+) and material-ui-confirm 3+? I've just updated [the demo](https://codesandbox.io/s/materialuiconfirm-demo-hzzdr) to also use Material UI 5 and setting the props works as expected.

**Comment by rickcaudill** [comment]
This does not work with typescript and MUI (5+). What I did was add a ts-ignore and sx parameters.

For example:
        cancellationButtonProps: {
            //@ts-ignore
            sx: {
                color: 'black',
            }
       }

**Comment by jonatanklosko** [comment]
I tried the demo with Typescript and it looks fine, the editor doesn't complain about `sx`, but if I add another unknown key it does complain. Not sure what may be the issue.

**Comment by kashua14** [comment]
> Hey @kashua14! Are you using the latest Material UI (5+) and material-ui-confirm 3+? I've just updated [the demo](https://codesandbox.io/s/materialuiconfirm-demo-hzzdr) to also use Material UI 5 and setting the props works as expected.

Yes, I am using the latest versions for both material UI 5 and material-ui-confirm.

However, I tested it in the code sandbox and it works as you say. So I think the problem might be with my application packages that may be conflicting with it.

@rickcaudill I will surely try that and give you feedback. 

Thank you a lot for your responses.

**Comment by jonatanklosko** [comment]
This is an old issue and there is already MUI v6, I'm going to close this. Feel free to open a new issue if you can reproduce issues in the demo.
